pkg/webhook/validate.go

```diff
@@ -26,10 +26,8 @@ const (
 	MlxMaxVFs  = 128
 )
 
-var (
-	nodesSelected     bool
-	interfaceSelected bool
-)
+var nodesSelected bool
+var interfaceSelected bool
```

Comment:
Wondering why we still have this here. Isn't the goal of the cleanup to not pass the value of this bool through a global variable? Instead, return it from the function?

Reply:
This is general cleanup. Earlier you requested this to be split into 2 PRs so that is what I did. This one needs to merged before the other PR gets merged.

Reply:
ok, sounds good then. Thanks!